<launch>
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node">
    <param name="video_device" value="/dev/video2" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="to_method" value="mmap" />
  </node>
  <!--
  <node name="image_view" pkg="image_view" type="image_view">
    <remap from="image" to="/usb_cam/image_raw" />
    <param name="autosize" value="true" />
  </node>
  -->
  <node name="face_detection" pkg="opencv_apps" type="face_detection">
    <remap from="image" to="/usb_cam/image_raw" />
    <param name="use_camera_info" value="false" />
    <param name="debug_view" value="false" />
    <param name="queue_size" value="1" />
    <param name="face_cascade_name" value="/usr/share/opencv/haarcascades/haarcascade_frontalface_alt.xml" />
    <param name="eyes_cascade_name" value="/usr/share/opencv/haarcascades/haarcascade_eye_tree_eyeglasses.xml" />
  </node>
  <node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" />
  <node name="receptionist_node" pkg="wqf7010_aa_prototype_development" type="receptionist_node.py" output="screen" />
</launch>
